File: benchmark/results/java-multilang/run-003/b-baseline/output/com/metaengine/demo/shipping/services/ShipmentService.java
package com.metaengine.demo.shipping.services;

import com.metaengine.demo.shipping.aggregates.Shipment;
import java.util.List;

/** ShipmentService domain service for the shipping domain. */
public class ShipmentService {
    /** Create a new Shipment from a partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Shipment by its identifier. May return null. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Shipments up to the given limit. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Shipment by its identifier. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-script-sonnet/run-001/b-baseline/output/src/domain/shipping/services/ShipmentService.ts
import { Shipment } from '../aggregates/Shipment';

/** ShipmentService service. */
export class ShipmentService {
  create(input: Partial<Shipment>): Shipment { throw new Error('not implemented'); }
  findById(id: string): Shipment | null { throw new Error('not implemented'); }
  list(limit: number): Shipment[] { throw new Error('not implemented'); }
  delete(id: string): void { throw new Error('not implemented'); }
}
